ios/browser/api/ai_chat/BUILD.gn

```diff
@@ -83,6 +84,8 @@ ios_objc_mojom_wrappers("ai_chat_mojom_wrappers") {
     "ConversationEntriesState",
     "ParentUIFrame",
     "Service",
+    "TabInformer",
```

Comment:
isn't this name changed now?

Reply:
Scary that this didn't break, but I guess that just means those interfaces compile on iOS? Nice catch :smile: